packages/terser/src/terser_minified.bzl

```diff
@@ -95,6 +95,12 @@ so that it only affects the current build.
     ),
 }
 
+def _filter_js(files):
+    return [f for f in files if f.is_directory or f.extension == "js" or f.extension == "mjs"]
+
+def _filter_sourcemap(files):
```

Comment:
nit: I don't feel like this function is carrying its weight, simple enough list comprehension to inline?

Reply:
valid point 🌮